python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1286,6 +1286,11 @@ def test_parquet_reader_v2(tmpdir, simple_pdf):
     assert_eq(cudf.read_parquet(pdf_fname), simple_pdf)
 
 
+def test_parquet_delta_byte_array(datadir):
```

Comment:
Should we have tests with skip_rows and/or nested columns? I saw some logic related to this in the kernels.

Reply:
So there's a can of worms. When I added the V2 headers, I modified a few roundtrip tests to run with both V1 and V2 headers. Now as we add more encodings, I think there really needs to be a lot more tests that run over a range of parameters, more like nvbench. But parquet already has the longest test to run and compile. Maybe this should be brought up for discussion elsewhere. 

Also, we can't use python to test because arrow doesn't support writing DELTA_BYTE_ARRAY yet (coming in version 14 IIRC), and we don't have an encoder ready yet either. I don't know if we can add a parquet file big enough to test all the weird edge cases.

Reply:
Agreed.
I would be okay with doing what we can with one test file and opening an issue for test coverage of new features.
My suggestion would be to have a test file that is a list of strings, and read it with and without skip_rows (and any other relevant option which I may be missing).

Reply:
Except #11480 :(

Reply:
We actually reverted the `skip_rows` here: https://github.com/rapidsai/cudf/pull/11657 and we now have arrow-14 support, would it make sense for now to add the tests?

Reply:
Thanks @galipremsagar! How did I miss _that_??? 

@vuule Do you think we have time to add skip rows tests now or just wait for DELTA_LENGTH_BYTE_ARRAY.

Reply:
> We actually reverted the skip_rows here: https://github.com/rapidsai/cudf/pull/11657

Oh, that only reverted the C++ side. The num_rows/skip_rows options are still missing from the python side of the house, which is what I have to use to generate the test files right now.

Another option is to use arrow-cpp for write and libcudf for read in the C++ tests.

FWIW I'm working on the DELTA_LENGTH_BYTE_ARRAY encoder/decoder right now, and will be able to test skip rows directly on the C++ side.

Reply:
Ah, my bad. Scratch that. Sorry for the confusion @etseidl ! This looks good to me then.